tests/scenario/conftest.py

```diff
@@ -0,0 +1,11 @@
+from unittest.mock import patch
+
+import pytest
+from charm import TraefikIngressCharm
+
+
+@pytest.fixture
+def traefik_charm():
+    with patch("charm.KubernetesServicePatch"):
+        with patch("lightkube.core.client.GenericSyncClient"):
```

Comment:
I think it's possible to
```suggestion
    with patch("charm.KubernetesServicePatch"), patch("lightkube.core.client.GenericSyncClient"):
```

Reply:
yup! I just learned that a couple of days ago and since then I've been refactoring them all over :)